From dc10d872c6688465565be3fd659ea75f667b4b9d Mon Sep 17 00:00:00 2001
From: Randy Li <randy.li@rock-chips.com>
Date: Wed, 12 Apr 2017 14:27:58 +0800
Subject: [PATCH] arm64: dts: rockchip: enable video decoder for RK3328 EVB

This commit would enable the VDPU and RKVDEC devices.
The VDPU works in the non combo mode.

Change-Id: I643350d5a2ac17759984fda2e95fb2b82701e7cf
Signed-off-by: Randy Li <randy.li@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328-evb.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-evb.dts b/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
index d85254a5f9f3..43bb09b01ede 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
@@ -381,6 +381,10 @@
 	};
 };
 
+&rkvdec {
+	status = "okay";
+};
+
 &sdio {
 	bus-width = <4>;
 	cap-sd-highspeed;
@@ -475,3 +479,6 @@
 	status = "okay";
 };
 
+&vpu_service {
+	status = "okay";
+};
-- 
2.35.3

